DockDoor/Views/Hover Window/WindowPreviewHoverContainer.swift

```diff
@@ -169,6 +171,15 @@ struct WindowPreviewHoverContainer: View {
         .padding(.top, (!previewStateCoordinator.windowSwitcherActive && appNameStyle == .popover && showAppTitleData) ? 30 : 0)
         .onAppear {
             loadAppIcon()
+            startLiveCapture()
+        }
+        .onDisappear {
+            Task {
+                await liveCapture.stopAllCaptures()
+            }
+        }
+        .onChange(of: previewStateCoordinator.windows) { _ in
```

Comment:
Is this call required? I feel like `onAppear` would cover every use case of this feature. 

Maybe it would be a better approach to make a more scoped version of `startLiveCapture` to take a single window id, and then we can attach the `onAppear` to the child WindowPreview's directly. This would cover if say, the coordinator had windows appended to it while the hover container is already visible. The child WindowPreview would call its own `onAppear` and would start the stream.